xml/installation-rhel-rhel_preinstall.xml

```diff
@@ -125,4 +125,10 @@ COMMIT
    support.
   </para>
  </section>
+ <section>
+  <title>PCI passthrough support not tested</title>
+  <para>
+   PCI pass-through has not been validated on &rhel; 
```

Comment:
This one sentence is a whole section :thinking: not that you can't do that, it just might look a bit funny when we build it. Is there a way to include this as a notation as a part of another section perhaps?

Although the section above is *very* similar, so it is consistent.

Reply:
Removed the section and added a note in installation-rhel-rhel_overview.xml